### PR TITLE
test: Add proper tests for empty deposits

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: test/TempusPool.Deposit.test.ts [hunk 1]
```diff
@@ -43,11 +43,18 @@ describeForEachPool("TempusPool Deposit", (pool:PoolTestFixture) =>
     );
   });
 
-  it.includeIntegration("Should revert on deposit 0", async () =>
+  it.includeIntegration("Should revert on depositing 0 BT", async () =>
```
Comment (RedFox20) [inline test/TempusPool.Deposit.test.ts:46]: BackingToken tests belong to TempusPool.DepositBackingToken.test.ts

Comment (axic) [reply test/TempusPool.Deposit.test.ts:46]: Hmm, actually there are many other BT-related cases, all of them should be moved. I would prefer to do that in a separate PR, maybe perhaps in #458.

**Review by RedFox20 — APPROVED** [review]
